pkg/skaffold/schema/latest/v1/config.go

```diff
@@ -568,6 +568,13 @@ type KubectlDeploy struct {
 	// DefaultNamespace is the default namespace passed to kubectl on deployment if no other override is given.
 	DefaultNamespace *string `yaml:"defaultNamespace,omitempty"`
 
+	// DryRun is the value kubectl will use with `kubectl create --dry-run=`. Valid values are:
```

Comment:
Would you mind fixing the schema enum collector to support an optional trailing `:`?
https://github.com/GoogleContainerTools/skaffold/blob/151b8d373d1eafcfe559fa1b302907c893cdda1a/hack/schemas/main.go#L52

Reply:
Maybe we should do this separately as it looks like there are a bunch of similar enums that aren't picked up.

Reply:
yup, I've fixed for this here for now by removing the `:`. I'll open up another PR to allow a trailing `:`

Reply:
https://github.com/GoogleContainerTools/skaffold/pull/6651